src/libs/H.Ipc.Generator/SourceGenerationHelper.cs

```diff
@@ -9,13 +11,26 @@ public static string GenerateClientImplementation(ClassData @class)
         return @$"
 #nullable enable
 
+using H.IpcGenerators;
```

Comment:
I used global:: instead of usings because this is the standard approach for Source Generators, which avoids build errors if the user has classes defined with names like in the System namespace, such as the Version class.
When using `using` this will throw an error saying the assembly cannot select a suitable class
When using global:: it will always use what you wrote

Reply:
Ah! I am fairly new to working with code generators, so this was an unknown to me. Will fix.